app/javascript/member-facing/backbone/action_form.js

```diff
@@ -65,33 +75,74 @@ const ActionForm = Backbone.View.extend({
     }
     this.$submitButton = this.$('.action-form__submit-button');
     this.buttonText = this.$submitButton.text();
+    this.existingMember = !!options.member.email;
```

Comment:
Do we find whether this is an existing member from whether the email field in the form has been pre-populated? Or what does this do?

Reply:
This was actually from @rodrei's PR: https://github.com/SumOfUs/Champaign/pull/1159/files#diff-2f530ef936f7a2a40165a04aa79e01c8R62, but the value passed to actionForm as `member` is `window.champaign.personalization.member`, which only gets populated if the member is recognised in the backend.